terraform/ci/main.tf

```diff
@@ -29,30 +33,55 @@ resource "google_project_service" "services" {
   disable_on_destroy = false
 }
 
-// IAM roles needed to run tests.
-resource "google_project_iam_member" "gh_access_acc_iam" {
-  for_each = toset(var.ci_iam_roles)
-
+resource "google_artifact_registry_repository" "artifact_repository" {
   project = var.project_id
 
-  role   = each.key
-  member = module.github_ci_infra.service_account_member
-}
+  location      = var.region
+  repository_id = var.registry_repository_id
+  description   = "Container registry for docker images."
+  format        = "DOCKER"
 
-module "github_ci_infra" {
-  source = "git::https://github.com/abcxyz/terraform-modules.git//modules/github_ci_infra?ref=46d3ffd82d7c3080bc5ec2cc788fe3e21176a8be"
+  depends_on = [
+    google_project_service.services["artifactregistry.googleapis.com"],
+  ]
+}
 
-  project_id = var.project_id
+// ci service account will need repoAdmin role to read, write and delete images
+resource "google_artifact_registry_repository_iam_member" "ci_service_account_iam" {
+  project = google_artifact_registry_repository.artifact_repository.project
 
-  name                 = "jvs"
-  github_repository_id = local.github_repo_id
-  github_owner_id      = local.github_owner_id
+  location   = google_artifact_registry_repository.artifact_repository.location
+  repository = google_artifact_registry_repository.artifact_repository.name
+  role       = "roles/artifactregistry.repoAdmin"
+  member     = format("serviceAccount:%s", var.ci_service_account_email)
 }
 
+
 module "jvs_common" {
   source = "../modules/common"
 
   project_id = var.project_id
 
   kms_key_location = var.kms_key_location
 }
+
+module "jvs_services" {
+  source = "../modules/jvs-services"
+
+  project_id = var.project_id
+
+  region          = var.region
+  service_ingress = "all"
```

Comment:
This is set to all because we are not using GCLB in CI project. Authentication is still required to send request to the service, so this shouldn't be a problem.